src/components/Button/button.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled from "@emotion/styled";
+import { keyframes } from "@emotion/core";
+
+interface ButtonProps {
+  text: string
+  type?: 'button' | 'submit' | 'reset'
+  isSubmitting?: boolean
+  isDisabled?: boolean
+}
+
+const Button = ({ text, type, isSubmitting, isDisabled }: ButtonProps) => (
+  <StyledButton type={type || 'submit'} role="button" aria-label={text} disabled={isDisabled}>
```

Comment:
```suggestion
  <StyledButton
    type={type || 'submit'}
    role="button" 
    aria-label={text} 
    disabled={isDisabled}
  >
```

Long line should be cascaded